UnitedIncome/components
# Display slider sticky hint correctly on first and last notches

### Description
If `tooltipStickyPosition` was `0` or `10`, then the hint would not show up correctly. Use separate styling rules for the hint and tooltip.

### Testing Instructions
1. Go to the `Slider` or `SliderCard` components in Storybook
2. Set the `tooltipStickyPosition` to `0`, `1`, and `10`
3. The hint and tooltip should display correctly
